starter-code/js/index.js

```diff
@@ -2,16 +2,25 @@ function deleteItem(e){
 
 }
 
-function getPriceByProduct(itemNode){
-
+// function getPriceByProduct(nodeItem)
+function getPriceByProduct(){
+  var priceItem = document.getElementsByClassName('.price-item');
+  priceItem = parseInt(priceItem[0].innerText); 
+  return priceItem;
 }
 
 function updatePriceByProduct(productPrice, index){
+  var subtotal = document.getElementsByClassName('subtotal');
+  subtotal = parseInt(subtotal[0].innerText);
 
+  var quantity = document.getElementsByClassName('quantity');
+  quantity = parseInt(quantity[0].value);
+  subtotal = getPriceByProduct() * quantity;
+  return subtotal;
```

Comment:
You don't need to return subtotal. You have to update the subtotal. 

It has to be somethig like:
```
document.querySelector('.subtotal').innerText = subtotal;
```